index.html

```diff
@@ -127,6 +127,16 @@ <h6 class="header3">About Myself</h6>
                 </nav>
                  
             </section>
+            <footer class="footer">
+                <h2 class="contact">Contact me</h2>
+                <p class="info">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                <form action="https://formspree.io/f/form_id" method="POST"> 
```

Comment:
- [ ] [REQUIRED] Please use your own `form_id` because your Formspree is not working. You need to read carefully [Formspree service](https://formspree.io/html/) again!

![image](https://user-images.githubusercontent.com/63915024/186448662-005de8c6-b869-400c-98a8-d2f54a9d723d.png)